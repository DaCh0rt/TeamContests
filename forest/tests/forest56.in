662 632
-2981 -9753
-9224 8339
-1871 -9536
8830 100
2788 5280
-8727 2796
-2034 -6752
4727 1628
-990 -1176
473 1969
-2694 -8655
-4356 3866
-9588 -5654
-3587 9479
7453 729
1663 -2277
-2315 4497
-5507 5544
7247 -403
9514 7420
-8874 -9253
284 8899
-6964 -6742
-2647 5959
-3741 9338
-9954 -3949
5604 -9836
-3462 -4721
-1723 -4275
-4782 -6429
2133 4168
-9140 1148
-6770 -2657
7908 -254
2514 -8599
6694 -8479
6570 -4185
-7535 1237
9504 -8344
8850 -3053
-330 -3382
-8309 1086
1851 3752
9203 7006
8366 8052
680 -4732
-1626 -4858
-5203 3494
2032 -2842
-5652 -4840
-6172 -8444
7506 1999
-5703 5924
-9980 6340
-8970 -5218
9652 -36
-8420 -364
-8505 -4791
9114 -9108
5716 -8082
-3034 -6911
-6584 -5638
5716 -924
-9145 6609
-8154 650
-292 -6410
-6217 1377
697 -8980
1028 6740
-9047 8479
-9790 -5421
7022 -4288
6790 -8283
3073 -4643
-6829 2961
9293 -4252
-7765 4937
5804 5581
-3320 -621
-7945 8924
-3276 -4400
5042 6540
1052 9988
-3095 2221
-5375 1078
-667 428
-6013 5836
467 3056
744 3705
8807 7466
-5684 -9041
-4621 7576
-2909 -7725
-4186 2399
-5045 -3311
979 -4341
-1880 -4387
9160 -8017
4847 2178
-3468 7377
9562 8902
4193 -8104
-5268 -5099
-2079 -6157
2707 1077
-3275 -931
4403 -4528
301 -305
3991 9869
-2384 -5393
254 432
3816 -8009
126 2292
2729 1743
8035 -1297
7948 -5685
-2131 4886
-67 -410
8770 9459
-3575 -7624
-7286 3390
-1048 -8216
3524 -2897
6598 5494
-3414 3225
-515 1779
4892 1502
-1529 4960
6078 -4182
-6752 734
-737 5394
-5023 1391
5364 6549
-9959 1496
5965 7279
-7122 5071
-2849 9142
-4899 6573
6403 1162
-2044 1091
3266 -4734
-9668 2730
5615 358
-5198 6152
0 1602
2069 -4361
3230 7746
6637 -2893
2481 -1794
-6810 760
-5281 -6531
7690 5740
-419 6310
-4998 973
6139 1293
2026 2094
765 -7221
6880 -8385
3486 -4890
2320 5759
-7702 -8702
5639 612
8481 7964
2567 -1390
4732 1174
8320 -3895
6210 546
-1547 1631
-1685 -5262
-1506 8830
-457 -6491
-9128 4535
1953 5145
8663 -5547
-6404 -9468
-2344 147
2512 1163
3750 -2004
2455 1476
-1964 -6820
6450 4784
1048 -9018
8749 -9311
-8186 -5391
-9255 2740
8675 1386
-3748 -5692
-6110 6120
-5257 -3090
-5297 -3932
6392 1176
-9267 9746
-1828 -9469
389 4756
-4318 1702
-9836 5184
7074 -8549
-5945 -5740
-4236 -2597
-6396 3259
-1061 -2940
6164 -4132
3514 -5020
-6470 -5406
6799 -6999
330 3420
9162 9254
2180 3659
-3903 -668
-2299 8817
-7743 -1918
-3531 -4200
2706 1698
-967 2540
-3162 -4930
4482 7372
-332 862
-8256 -7642
3422 -5754
-5190 -4959
9411 2558
-8299 -4274
97 -6719
-2287 -2328
9445 -2988
6526 -8168
-9353 7491
5792 -6598
-9763 -2326
3682 -2391
-5875 691
2054 6620
9733 7862
4783 4341
2803 -2489
967 6326
2119 8662
-2071 -2680
-2969 6435
1443 1114
3101 -410
5391 -3634
5523 9746
-4402 -246
1342 -1459
6435 -126
7324 -8134
5482 -7554
-6035 -258
-2851 -7712
4275 -4696
3185 -8855
-7242 4881
8292 2308
9963 -6791
-3440 -3204
8648 2208
4409 9668
5833 -8423
5908 -6728
1977 9295
-6037 3091
6353 -9419
-9672 4186
7199 -9043
-1056 8156
-3760 -9851
-539 -6255
6073 -2117
6467 -354
-3449 5942
2008 -3543
-4774 9533
2270 3257
9611 2906
5312 5594
9114 4851
2525 -5661
-1793 -7454
-7783 5598
-1051 7869
4909 -7625
8251 3458
-5781 -3416
-1568 8940
-4113 1593
8280 8485
1217 -4248
-2048 400
-8303 -5590
-5735 -6669
5908 -9847
9788 -4695
-4575 -3607
3431 -8621
7855 1841
6021 3200
6322 -3017
6892 -1809
-3605 -7713
-8130 3876
4145 9195
-7433 5417
5585 2944
-7635 157
-6619 -7217
2084 -7604
-1625 -6970
-4473 -1604
-2460 -9828
4676 8980
-3760 -1599
9992 -6983
-8415 -8992
-5519 -8799
-7481 1911
-8848 -2005
-9369 697
425 -8417
-8514 -6809
1549 8712
-82 -5859
2513 8628
7402 2810
385 6241
-530 -6214
8280 -5893
3485 -6033
816 -7322
7103 3456
-1158 -9888
791 9446
3775 7598
9801 851
1068 -9362
5016 -2185
-2655 -6640
8862 2709
-3448 -7436
3537 -1802
-1409 3791
3991 -2056
9657 4215
-693 3974
-9300 3501
-5086 -2808
-6463 8023
1993 -2888
-2496 -4706
5835 3719
1568 468
-4100 8877
3980 2235
6146 -2096
8543 -8662
-1252 -8279
-1153 2101
2041 -7382
-5274 3387
-9720 -7813
-1266 -1957
7990 5710
-8944 9565
9593 7222
6694 7391
-3898 -9825
885 6816
1131 -7559
-3627 -5224
-2236 5347
-5475 3996
3896 -4907
8549 9865
8633 -4473
-5667 4794
6942 -1296
1669 -8831
-1096 -3298
7429 1906
4293 7488
-1946 -6797
-9067 -3172
4936 5426
-7637 5728
-4259 -5297
-272 7333
-2833 -6015
-4265 -7436
5888 -23
-5966 496
-8860 247
8564 -4262
-9405 3308
6074 5199
-3845 -2289
1181 -2107
7676 4218
3523 -5668
9677 -7186
2441 -4145
4712 -3112
4619 -5240
9496 9823
-5993 -5047
-5152 2478
5152 7933
-7623 1478
9004 6793
9149 -4334
2191 9816
-6692 -6331
8458 -751
4771 3452
1602 -2817
5613 6199
-9104 8237
1672 -9751
-8275 7411
-2119 -6684
538 -4271
7708 8558
-8950 -4890
-2083 -3051
9315 5221
-5076 -3425
7180 3969
-2074 -8071
-6008 6048
8869 -9253
-5692 4160
88 6801
8271 -7689
7049 -6545
8704 -1816
2907 -389
-2326 -9237
2411 398
3243 -5514
-396 7712
9890 7601
9802 -9913
7716 -7302
1651 -1807
9871 8309
-1122 674
-5580 6960
5045 -2298
2579 -6799
7190 9754
5508 7999
7600 -1225
7735 2464
3115 6811
-9225 1220
8019 -5411
-985 8103
5029 6415
9641 4878
7316 5131
-7909 3816
2861 -297
-969 -5621
-8185 -6926
6425 1189
-2504 2570
3913 8105
-634 5275
2657 -1512
2615 3099
-5185 7064
1193 -8356
-4169 -243
7008 8910
286 -65
3232 778
1325 1558
7503 8747
-2656 7166
-3191 9647
2895 5543
-2300 -104
-8080 -7505
-8036 -7446
902 7524
-1665 -1782
5720 8023
-2990 -7475
1908 -2403
-367 6659
-7815 4399
1193 1982
1325 7724
5068 -6526
-6513 8615
4973 -4943
-354 -7219
5499 -5486
-8518 3302
-1891 -634
-5757 -322
-6942 -5490
1328 -7599
2349 -6066
-4160 -7398
5312 -2926
-178 -6508
2648 2433
3267 -7952
-279 4695
-7504 -1786
2364 -6981
292 -2031
1892 -3957
-77 -7441
-3754 6385
7292 8031
-5376 -927
-1788 9852
-8736 -6280
-2956 5485
-5374 7747
3892 3134
9456 -6685
7035 4766
8766 2306
-3899 -9951
7880 -2162
6936 5100
2742 5651
-6488 7662
387 -7668
-9077 166
-7525 -5655
-1270 -9277
6625 3163
-3571 -9699
-963 1487
-767 -901
2345 -5626
-8448 4435
4147 8326
6183 -2078
9291 -7188
-784 -330
-3171 -3475
-7075 -1772
4876 6903
-187 -9438
-8090 2941
-4864 -2916
2172 303
1167 7499
-8741 1222
5169 -9299
2918 4855
-540 970
2987 4193
3986 -3050
-614 -6278
-5790 -3116
9586 3888
7929 5869
6463 8946
-4721 -1083
-6557 4816
-7443 3403
1162 -7556
-1799 -772
-8538 6953
-1437 4949
2934 -9353
1922 -4594
1830 4353
-8023 7147
3865 -3264
-3942 -2561
-9001 5425
-349 -1556
9224 -5186
9682 3518
-5858 -994
8112 -5755
9121 -4375
7524 -127
8453 -5016
-2108 2268
-2672 442
-4625 2801
3096 -3754
-9337 -4817
-1792 9840
4246 -4938
3107 2170
-7466 1436
-9695 -1290
-2664 4272
4308 3234
-7821 6294
-6983 -2091
6429 2403
1182 -3758
7331 987
-6111 1268
-2887 -8386
787 7383
-9312 2695
2776 921
-1023 1634
5562 -8700
1547 3414
-6284 9262
-378 -1549
-4162 1414
2643 8712
-5217 -6983
-8948 9560
-5924 2390
438 -3401
-3606 -1441
2696 -3576
-2609 -3260
-5536 -1002
-6891 -2811
-8672 -185
2148 -9192
-6498 -5298
-1278 -8722
-9033 2192
6020 6447
-7966 -8820
2843 5977
-1235 -1852
-2315 8660
-1129 7612
465 -9217
8033 252
3551 8353
-1077 3339
-6569 -1481
86 7419
-364 -4824
4226 -3903
2240 7705
2463 -5340
-8121 -8834
3937 -4835
-4389 -3424
1133 4744
-1577 7264
-3958 -2917
758 -729
-4405 195
-9646 3489
-9220 9400
660 -9513
9958 -976
-3945 -399
5556 2495
105 -7405
3507 1163
7376 -8869
-5969 -5239
