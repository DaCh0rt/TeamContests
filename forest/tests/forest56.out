80562.12 438612135.25
